packages/loader/container-loader/src/container.ts

```diff
@@ -1682,8 +1685,19 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         switch (outboundMessageType) {
             case MessageType.Operation:
             case MessageType.RemoteHelp:
-            case MessageType.Summarize:
                 break;
+            case MessageType.Summarize: {
+                // todo this is only needed for staging so the server
+                // know when the protocol tree is included
+                // this can be removed once all clients send
+                // protocol tree by default
+                const summary = contents as ISummaryContent;
```

Comment:
Maybe reference an issue tracking removal? That said, given it's at the loader layer, we should do deprecations here on a year+ timeframe.... And likely leveraging ODSP + FRS telemetry and some mechanism to disallow very old loaders to be booted in the first place (i.e. in 2 years, even if someone using this version of loader, it will stop working, but will stop working in some well-planned manner)